CONTRIBUTING.md

```diff
@@ -12,34 +12,33 @@
 </p>
 
 ## Table of contents
+- [Table of contents](#table-of-contents)
 - [Commits](#commits)
 - [Pull Requests](#pull-requests)
 
 
 ## [](#commits)Commits
 
-- All commits may start with an uppercase letter
-- All commits must be written in present tense
-- All commits may be multiline commits
-- The commit headline must have a maximum of 71 chars
-- The commit message must have a maxmum of 71 chars per line
-- If possible, include link to online topic as reference (Ref: )
-- Detailed explanation why the commit was done
-- Include reference to issue if one exists
-- All commits must be verified
-- No commits, under no circumstances, should be directly pushed to master (deadly)
-- Each commit should only serve one purpose
+-   All commits may start with an uppercase letter
+-   All commits must be written in present tense
+-   All commits may be multiline commits
+-   The commit headline must have a maximum of 71 chars
+-   The commit message must have a maximum of 71 chars per line
+-   If possible, include link to online topic as reference (Ref: )
+-   Detailed explanation why the commit was done
+-   Include reference to issue if one exists
+-   All commits must be verified
+-   No commits, under no circumstances, should be directly pushed to master (deadly)
+-   Each commit should only serve one purpose
 
 ## [](#pull-requests)Pull Requests
 
-- Each pull request should only follow one topic
-- You must follow the guidelines / template
-- Each pull request must answer one more more issues
-- Each pull request must have at least one reviewer
-- Each pull request must have at least one assigned developers
-- Each pull request must have a label
-- Each pull request should be merged by a reviewer
-
+-   Each pull request should only follow one topic
+-   You must follow the guidelines / template
+-   Each pull request must have at least one reviewer
+-   Each pull request must have at least one assigned developers
```

Comment:
```suggestion
-   Each pull request must have at least one assigned developer
```

Reply:
I have already dealt with the same issues in my snek repos.

Reply:
I copied it from `snek-at/ops-front`.